cluster-up-example

```diff
@@ -18,7 +18,13 @@ oc_password=developer
 #Digger Variables
 digger_project_name=digger
 digger_platforms="android,ios"
-
+#proxy_url=127.0.0.1
```

Comment:
the proxy url should have the following syntax
"https://<user>:<pass>@<host>:<port>" - with auth
or
"https://<host>:<port>"
We should be explicit about this in the example to make things as easy as possible for the user